test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -111,6 +120,13 @@ def generate_baseline(prototypes_generated, id_prototype_mapping)
       # Initialize Standard class
       @prototype_creator = Standard.build('90.1-PRM-2019')
 
+      # user data CSV files are in @user_data_dir, if appicable
+      # user data JSON files will be created in sub-folder inside @test_dir
+      model_name = mod.empty? ? "#{building_type}-#{template}-#{climate_zone}" : "#{building_type}-#{template}-#{climate_zone}-#{mod_str}"
+      proto_run_dir = "#{@test_dir}/#{model_name}"
+      json_path = @prototype_creator.convert_userdata_csv_to_json(@user_data_dir, proto_run_dir)
+      @prototype_creator.load_userdata_to_standards_database(json_path)
```

Comment:
Do we want to test that the data has been load correctly?

Reply:
I ran a temporary test to make sure it works. We will add permanent tests when we get to JIRA tasks that access the user data.